libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset.service.ts

```diff
@@ -19,15 +19,13 @@ export class SkyVerticalTabsetService {
 
   public animationTabsVisibleState: string;
 
-  public set content(value: ElementRef) {
-    this._content = value;
-  }
+  public content: ElementRef | undefined;
```

Comment:
Is our pattern for public API properties in our services to not use getters and setters?

Reply:
Not necessarily - but in this case the only thing the setter was doing was hiding the `content` from being able to be grabbed outside of the class. But this class is internal and so we will be ok to just have it be a normal field. The setter wasn't doing anything else here.